### ISSUE TITLE
Using chef-ingredient fails with uninitialized constant Chef::Resource::ChefIngredient::DefaultHandler

### ISSUE THREAD

**Issue by smith** [chef-cookbooks/chef-ingredient#140]
### Cookbook version
2.0.0

### Chef-client version
12.12.13

### Platform Details
Ubuntu 12.04 in EC2

### Scenario:
Attempt to converge a recipe that uses a `chef_ingredient` resource

### Steps to Reproduce:
A recipe like:

```ruby
chef_ingredient "automate"
```

### Expected Result:
The ingredient is installed

### Actual Result:

```
              ================================================================================
              Recipe Compile Error in /var/chef/cache/cookbooks/chef-ingredient/resources/chef_ingredient.rb
              ================================================================================
              
              NameError
              ---------
              uninitialized constant Chef::Resource::ChefIngredient::DefaultHandler
              
              Cookbook Trace:
              ---------------
                /var/chef/cache/cookbooks/chef-ingredient/resources/chef_ingredient.rb:59:in `block in class_from_file'
                /var/chef/cache/cookbooks/chef-ingredient/resources/chef_ingredient.rb:54:in `class_from_file'
                /var/chef/cache/cookbooks/compat_resource/files/lib/chef_compat/monkeypatches/chef/resource/lwrp_base.rb:50:in `build_from_file'
              
              Relevant File Content:
              ----------------------
              /var/chef/cache/cookbooks/chef-ingredient/resources/chef_ingredient.rb:
              
               52:  platform_family = node['platform_family']
               53:  
               54:  action_class do
               55:    include ChefIngredientCookbook::Helpers
               56:  
               57:    case platform_family
               58:    when 'debian', 'rhel', 'suse', 'windows'
               59>>     include ChefIngredient::DefaultHandler
               60:    else
               61:      # OmnitruckHandler is used for Solaris, AIX, FreeBSD, etc.
               62:      # Eventually, we would like to support all platforms with the DefaultHandler
               63:      include ChefIngredient::OmnitruckHandler
               64:    end
               65:  end
               66:  
               67:  action :install do
               68:    check_deprecated_properties
              
              Platform:
              ---------
              x86_64-linux
              
              [2017-03-24T19:37:56+00:00] ERROR: Running exception handlers
              [2017-03-24T19:37:56+00:00] ERROR: Exception handlers complete
              [2017-03-24T19:37:56+00:00] INFO: Sending resource update report (run-id: 55c3c257-f209-4498-a000-c2bf8f614fdf)
              [2017-03-24T19:37:56+00:00] FATAL: Stacktrace dumped to /var/chef/cache/chef-stacktrace.out
              [2017-03-24T19:37:56+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              [2017-03-24T19:37:56+00:00] ERROR: uninitialized constant Chef::Resource::ChefIngredient::DefaultHandler
              [2017-03-24T19:37:57+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
```

**Comment by smith** [comment]
Was unable to reproduce this in Test Kitchen with the versions of chef-client and Ubuntu shown in the issue.

**Comment by schisamo** [comment]
This was fixed in #143
